plugins/woocommerce/client/admin/client/settings-payments/components/other-payment-methods.tsx

```diff
@@ -121,35 +166,38 @@ export const OtherPaymentMethods = () => {
 		return null;
 	}
 
-	const hasWcPaySetup = wcPayGateway.enabled && ! wcPayGateway.needsSetup;
-
 	return (
-		<>
-			<Button
-				className="is-tertiary"
-				href="https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/?utm_source=payments_recommendations"
-				target="_blank"
-				value="tertiary"
-				rel="noreferrer"
-			>
-				<span className="other-payment-methods__button-text">
-					{ hasWcPaySetup
-						? __(
-								'Discover additional payment providers',
-								'woocommerce'
-						  )
-						: __(
-								'Discover other payment providers',
-								'woocommerce'
-						  ) }
-				</span>
-				<ExternalIcon size={ 18 } />
-			</Button>
-			{ additionalGateways.length > 0 && (
-				<AdditionalGatewayImages
-					additionalGateways={ additionalGateways }
-				/>
-			) }
-		</>
+		<Panel className="other-payment-methods">
+			<PanelBody title="Other payment methods">
+				<div className="other-payment-methods__grid">
+					{ mockOtherPaymentMethods.map(
+						( gateway: PaymentGateway ) => (
+							<div
+								className="other-payment-methods__grid-item"
+								key={ gateway.id }
+							>
+								<img src={ gateway.image_72x72 } alt="" />
```

Comment:
Similar comment about adding alt text

Reply:
fixed in https://github.com/woocommerce/woocommerce/pull/52030/commits/0444c8c78b346bc2b8061a88e4e8898d5f3357f7